frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -10,9 +14,23 @@ type MenuItem = {
   path: string
 }
 
+const blankProfile =
+  'https://st3.depositphotos.com/4111759/13425/v/600/depositphotos_134255626' +
+  '-stock-illustration-avatar-male-profile-gray-person.jpg'
```

Comment:
try not to use resources that designers didn't create, use this svg instead
https://we.tl/t-7kEeDH0qvN